### PR TITLE
Add JFR_SUPPORT Flag

### PR DESCRIPTION
**Pull request by yathamravali**
JFR Reflection code added within JFR_SUPPORT is not getting invoked. Hence opening this PR to add flag into JPP_TAGS

Signed-off-by: Ravali Yatham <rayatha1@in.ibm.com>

### CODE REVIEW

**Comment by yathamravali** [comment]
@tajila - Please review

File: closed/JPP.gmk [hunk 1]
```diff
@@ -38,6 +38,10 @@ ifeq (true,$(BUILD_OPENJCEPLUS))
   JPP_TAGS += OPENJCEPLUS_SUPPORT
 endif # BUILD_OPENJCEPLUS
 
+ifeq (true,$(OPENJ9_ENABLE_JFR))
+  JPP_TAGS += JFR_SUPPORT
+endif # OPENJ9_ENABLE_JFR
+
```
Comment (keithc-ca) [inline closed/JPP.gmk:44]: This should be added before line 37 like newer versions. A change should be proposed and merged for jdk17 before this is considered.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: closed/JPP.gmk (hunk 1 above)
Comment (yathamravali) [reply closed/JPP.gmk:40]: ok, Opened https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/390

Comment (tajila) [reply closed/JPP.gmk:40]: Please add for JDK21, JDK23 and JDKNext as well

Comment (keithc-ca) [reply closed/JPP.gmk:40]: The `JFR_SUPPORT` tag is already supported in jdk21 and newer versions.

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
I will create a separate pull request to fix the order as requested in https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/811#discussion_r1783194153.

**Pull request merged**